src/wizard/ionic/index.md

```diff
@@ -94,8 +92,12 @@ Sentry.init(
     // We recommend adjusting this value in production.
     tracesSampleRate: 1.0,
     integrations: [
-      new BrowserTracing({
-        tracingOrigins: ["localhost", "https://yourserver.io/api"],
+      new SentrySibling.BrowserTracing({
+        // Set `tracePropagationTargets` to control for which URLs distributed tracing should be enabled
+        tracePropagationTargets: [
```

Comment:
Should the wizzard also have the `routingInstrumentation` parameter?

Reply:
If our instrumentation doesn't work correctly (see comment/question above), I'd rather not add it here.